### ISSUE TITLE
Survival data

### ISSUE THREAD

**Issue by lmwaniki** [chap-uzima/DataWarehouse#13]
@chris-githu
CDC presentations are round the corner, in preparation kindly pull for me survival data.

Thanks

Lawrence

**Comment by lmwaniki** [comment]
[variables.xlsx](https://github.com/chap-uzima/DataWarehouse/files/2222297/variables.xlsx)

**Comment by chris-githu** [comment]
@lmwaniki 
Sent via email

**Comment by lmwaniki** [comment]
Hi Chris,

Our list of Active patients has 265 preART- we did a sample for Comboni and it seems this patients are care ended with no new clinical encounter form.

**Comment by lmwaniki** [comment]
Hi Chris,
Guess the query is also dropping part of the regimen
Mater Patient ID 7224 screenshot attached...
![regimen](https://user-images.githubusercontent.com/34023464/43461027-0db8ef90-94db-11e8-870e-92a0760c02da.JPG)

**Comment by chris-githu** [comment]
@lmwaniki We are not dropping the regimen parts. 

However, Mater has an issue with this particular regimen.
@vogoti @mkaburu 
Please see the snippet below
FDC `Tenofovir/Lamivudine/Efavirenz (TDF/3TC/EFV) 300/300/600mg FDC Tabs` with `drug_pk=1317`
has not been mapped correctly in the generic parts table.

You should have either
- 3 records in `lnk_druggeneric` linking to `TDF`, `3TC` and `EFV` in `mst_generic`
- Or 1 record in `lnk_druggeneric` linking to `TDF-3TC-EFV` in `mst_generic`


![mater](https://user-images.githubusercontent.com/13725334/43505092-74a08784-956e-11e8-860b-265bda2753f4.PNG)

Let me know if you need my assistance with this.

**Comment by mkaburu** [comment]
@chris-githu Yes i think support would be good since this affects some other sites

**Comment by lmwaniki** [comment]
@chris-githu 
Kindly add a variable "care end date" to assist in trends comparisons across various Quarters

kind regards

Lawrence

**Comment by lmwaniki** [comment]
Hi @chris-githu 
A kind remind please add "care end date" variable to the survival dataset to assist in running attrition trends

regards

Lawrence

**Comment by lmwaniki** [comment]
Hi @chris-githu 
Kindly requesting the survival data as of Dec 2018
regards

Lawrence

@irenefaith @vogoti
